services/convert/repository.go

```diff
@@ -183,6 +183,7 @@ func innerToRepo(ctx context.Context, repo *repo_model.Repository, mode perm.Acc
 		DefaultBranch:                 repo.DefaultBranch,
 		Created:                       repo.CreatedUnix.AsTime(),
 		Updated:                       repo.UpdatedUnix.AsTime(),
+		ArchivedAt:                    repo.ArchivedUnix.AsOptionalTime(),
```

Comment:
I guess this `AsOptionalTime` is not the same behavior as other "time" related fields in Gitea.

In history, Gitea uses "0" for no-time. But `AsOptionalTime` would get null.

If there is no strong reason, should we keep the old behavior for consistency?

Reply:
If really causes inconsistency. Before all clients already use  "1970-01-01" for no-date. Now, the client sometimes needs to process null, sometimes needs to process  "1970-01-01" .....

Unless we have a full plan to refactor all date to support null.